extensions/crypto/01_crypto.js

```diff
@@ -112,7 +112,13 @@
         context: "Argument 2",
       });
 
-      data = data.slice(0);
+      if (ArrayBuffer.isView(data)) {
+        data = new Uint8Array(data.buffer, data.byteOffset, data.byteLength);
+      } else {
+        data = new Uint8Array(data, data.byteOffset, data.byteLength);
```

Comment:
```
> new ArrayBuffer().byteOffset
undefined
```

I think the second and third arg to the constructor can go.

Reply:
Oh, this is why you don't review commits in isolation (unless it makes sense)

Reply:
> Oh, this is why you don't review commits in isolation (unless it makes sense) 🤦

Hehe apologies, I pushed immediately rather than taking a minute to think 😁